CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 # OMC3 Changelog
 
-#### 2024-09-18 - v0.16.0 - _jdilly_
+## IN PROGRESS - v0.16.1 - _fsoubelet_
+
```

Comment:
I left it #### in the past because I like the format of how it looks better ... but if you prefer it this way, or are more bothered by the IDE warning than how the log looks like, this is also ok

Reply:
No strong opinions, I tend to make the linters happy but it's not like it's a big deal, it's barely code.